src/backend/marsha/development/views.py

```diff
@@ -75,15 +76,25 @@ def get_context_data(self, **kwargs):
                 "oauth_signature_method": "",
             }
 
+        email = fake.email(safe=False)
+
         return {
             "domain": domain,
-            "lis_result_sourcedid": fake.user_name(),
+            "resource_link_id": fake.bs(),
+            "user_id": fake.md5(),
+            "lis_result_sourcedid": email.split("@")[0],
+            "lis_person_contact_email_primary": email,
             "uuid": uuid.uuid4(),
             "select_context_id": playlist.lti_id,
             "select_content_item_return_url": self.request.build_absolute_uri(
                 reverse("development:lti-development-view")
             ),
             "oauth_dict": oauth_dict,
+            "last_objects": {
+                "videos": Video.objects.all()[:5],
+                "documents": Document.objects.all()[:5],
+                "meetings": Meeting.objects.all()[:5],
```

Comment:
Don't you need some `.order_by` to actually get the "latest" objects ? Order by PK or `updated_on`, I don't know ^^

Reply:
Of course, you're right! Thanks :)